app/packages/core/src/plugins/SchemaIO/components/SliderView.tsx

```diff
@@ -79,17 +74,18 @@ export default function SliderView(props) {
     multipleOf: schemaMultipleOf,
     view,
   } = schema;
+
   const {
     min: viewMin,
     max: viewMax,
-    value_format: valueFormat = "%",
```

Comment:
I feel like we should keep the option to define the valueFormat from the Python context still. Because we don’t want to come back and change all this later, since we are now defining this outside of Python’s control as a const in React

Reply:
makes sense